save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/OrganizationController.kt

```diff
@@ -70,19 +70,28 @@ internal class OrganizationController(
 ) {
     private val webClientToPreprocessor = WebClient.create(config.preprocessorUrl)
 
+    /**
+     * @param includeDeleted whether deleted organizations should be included in
```

Comment:
@sanyavertolet forces to use swagger api annotation instead of kdoc for API endpoints

Reply:
Addressed with 1ad377dbb7b516ab572c629fb5f184d50091a605.